app/keepers.go

```diff
@@ -38,17 +38,17 @@ import (
 	evidencekeeper "github.com/cosmos/cosmos-sdk/x/evidence/keeper"
 	evidencetypes "github.com/cosmos/cosmos-sdk/x/evidence/types"
 
-	// Governance: Allows stakeholders to make decisions concering a Cosmos-SDK blockchain's economy and development
+	// Governance: Allows stakeholders to make decisions concering a Cosmos-SDK blockchain's economy and development.
 	govkeeper "github.com/cosmos/cosmos-sdk/x/gov/keeper"
 	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
-	paramproposal "github.com/cosmos/cosmos-sdk/x/params/types/proposal"
 
-	// Params: Parameters that are always available
+	// Params: Parameters that are always available.
 	"github.com/cosmos/cosmos-sdk/x/params"
 	paramskeeper "github.com/cosmos/cosmos-sdk/x/params/keeper"
 	paramstypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	paramproposal "github.com/cosmos/cosmos-sdk/x/params/types/proposal"
 
-	// Slashing:
+	// Slashing:.
```

Comment:
lolwat

Reply:
in fact, this explains a half dozen things.  

thanks!